src/Workspaces/Core/Portable/TemporaryStorage/TrivialTemporaryStorageService.cs

```diff
@@ -21,6 +21,7 @@ private TrivialTemporaryStorageService()
 
     public ITemporaryStorageStreamHandle WriteToTemporaryStorage(Stream stream, CancellationToken cancellationToken)
     {
+        stream.Position = 0;
```

Comment:
if there's a bug here, i'm curious how any VB to C# (or vice versa) tests pass today :)

Reply:
Debugging through the test on Linux, I can definitely see the diagnostic `error BC31519: '{0}' cannot be referenced because it is not a valid assembly.` being there, I guess the test just doesn't verify diagnostics. I'm not sure how the go-to-def then works, I guess it doesn't really need the metadata reference, but works differently under the hood?